src/containers/homepage/homepage.container.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Layout } from 'antd';
+import { SurveyListComponent } from './survey-list.component';
+
+const { Header, Footer, Sider, Content } = Layout;
+
+export class HomepageContainer extends Component {
+    render() {
+        const data = {
```

Comment:
Ideally, I would use the "Single source of truth" concept here and get this data from the store.. How would I go about putting it there, and how do I retrieve it in any meaningful way?

Reply:
Gotta create the required reducer/actions for this, & map that state to this container's props